README.md

```diff
@@ -0,0 +1,69 @@
+
+Supports Linux and Windows based systems with package support for Debian, Redhat,
+and Windows.
+
+---
+
+Resources
+---------
+
+### splunk_install
+Manages an installation of Splunk
+
+##### Action *:install*
+Installs Splunk or Universal Forwarder.
+
+Properties:
+
+| Name           |               Type(s)               |       Required       | Default                                  | Description                                                                                                                                                                                                                                                            |
+|:---------------|:-----------------------------------:|:--------------------:|:-----------------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| package        | `:splunk` or `:universal_forwarder` |          No          | `:splunk`                                | Specifies the Splunk package to install. If the name of the resource is `'splunk'` or `'universal_forwarder'`, then it will infer which package you intend to install.                                                                                                 |
+| version        |               String                |       **Yes**        |                                          | Version of Splunk to install                                                                                                                                                                                                                                           |
+| build          |               String                |       **Yes**        |                                          | Build number of the version                                                                                                                                                                                                                                            |
+| base_url       |               String                |          No          | `'https://download.splunk.com/products'` | Base url to pull Splunk packages from. Use this if you are mirroring the downloads for Splunk packages. The resource will append the version, os, and filename to the url like so: `{base_url}/splunk/releases/0.0.0/linux/splunk-0.0.0-a1b2c3d4e5f6-Linux-x86_64.tgz` |
+| accept_license |                true                 | **Yes (on Windows)** |                                          | Set if you accept Splunk's license agreement on install. Currently only for the Windows Package, as it requires the license to be accepted at install time. On Splunk for Linux, it can be accepted on first run with `--accept-license`.                              |
```

Comment:
I've been thinking about this a lot... It's a parameter, that's required to be set to true in a package designed for automation. What if we removed this parameter, and instead put something to the effect of "Using resources of this cookbook to install and run Splunk means you agree to Splunk's EULA packaged with the software, online at http://www.splunk.com/en_us/legal/splunk-software-license-agreement.html" Then we can get rid of the option, and the logic reacting to having different options for it.